generators/server/templates/src/main/java/package/config/DatabaseConfiguration.java.ejs

```diff
@@ -174,7 +175,7 @@ public class DatabaseConfiguration {
     @Bean(initMethod = "start", destroyMethod = "stop")
     @Profile(JHipsterConstants.SPRING_PROFILE_DEVELOPMENT)
     public Object h2TCPServer() throws SQLException {
-        String port = "1" + env.getProperty("server.port");
+        String port = String.valueOf(SocketUtils.findAvailableTcpPort());
         log.debug("H2 database is available on port {}", port);
```

Comment:
maybe we should make this logging from debug to info to print the port

Reply:
That makes sense. Well it is useless on anyway since we proxy the request.